.github/workflows/test.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         rubygems_version: ['3.3.11', 'latest']
         ruby_version: ['3.1.2']
     name: Rails tests (Ruby ${{ matrix.ruby_version }}, RubyGems ${{ matrix.rubygems_version }})
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
     env:
```

Comment:
We should probably run our test on same image as prod.

Reply:
There is no alpine github image, the only way would be to run tests in docker image itself (which is possible, but slower, I can setup that for master push only for example). This is just to keep the current CI running (get rid of deprecation).